src/util/amount.rs

```diff
@@ -955,6 +965,45 @@ pub mod serde {
         }
     }
 
+    pub fn parse<'d, D, A>(d: D) -> Result<A, D::Error>
+    where
+        D: Deserializer<'d>,
+        A: SerdeAmount,
+        <A as FromStr>::Err: ::std::fmt::Display,
+    {
+        use ::std::fmt::{self, Display, Formatter};
+        use ::std::marker::PhantomData;
+        use ::serde::de;
+
+        struct Visitor<ValueT>(PhantomData<ValueT>);
+
+        impl<'de, ValueT> de::Visitor<'de> for Visitor<ValueT>
+        where
+            ValueT: FromStr,
+            <ValueT as FromStr>::Err: Display,
+        {
+            type Value = ValueT;
+
+            fn expecting(&self, formatter: &mut Formatter) -> fmt::Result {
+                formatter.write_str("a Bitcoin amount")
```

Comment:
Maybe note that it expects a denomination?

Reply:
done